docassemble/ALToolbox/display_template.py

```diff
@@ -0,0 +1,39 @@
+import re
+from .copy_button import *
+
+def display_template(template, scrollable=True, collapse=False, copy=False, class_name=None):   
```

Comment:
These names look good to me--easier to read/use than `copiable` and `collapsible` and I think easy to understand. Highlighting in case @plocket  or @BryceStevenWilley have more thoughts.